python/plugins/processing/algs/lidar/fusion/Cover.py

```diff
@@ -87,7 +88,7 @@ def processAlgorithm(self, progress):
             FusionUtils.createFileList(files)
             commands.append(FusionUtils.tempFileListFilepath())
         FusionUtils.runFusion(commands, progress)
-        commands = [os.path.join(FusionUtils.FusionPath(), 'DTM2TIF.exe')]
+        commands = [os.path.join(FusionUtils.FusionPath(), 'DTM2ASCII.exe')]
```

Comment:
you're right, but within FUSION .tif files are limited to the grey scale: in this way, all the values are related to this scale and not kept as real values (e.g. in a canopy height model, values from 0.15-23.68 metres are shifted to 0-159).